pkg/network/usm/sharedlibraries/watcher_test.go

```diff
@@ -357,8 +358,6 @@ func (s *SharedLibrarySuite) TestSoWatcherLeaks() {
 		return checkPathIDDoesNotExist(watcher, fooPathID1) && checkPathIDDoesNotExist(watcher, fooPathID2)
 	}, time.Second*10, time.Second, "")
 
-	checkWatcherStateIsClean(t, watcher)
```

Comment:
I originally did it because the test assertions where way to coupled with the internal state of `FileRegistry`, and we are no longer exposing this.
Having said that, I think I should probably preserve it while I don't have good test coverage for the `utils` package, so I re-added the check